selfdrive/car/honda/carstate.py

```diff
@@ -98,24 +96,13 @@ def get_can_signals(CP):
                 ("WHEELS_MOVING", "STANDSTILL", 1)]
     checks += [("DOORS_STATUS", 3)]
 
-  if CP.carFingerprint == CAR.CIVIC:
-    signals += [("CAR_GAS", "GAS_PEDAL_2", 0),
-                ("MAIN_ON", "SCM_FEEDBACK", 0),
-                ("EPB_STATE", "EPB_STATUS", 0),
-                ("BRAKE_HOLD_ACTIVE", "VSA_STATUS", 0)]
-  elif CP.carFingerprint == CAR.ACURA_ILX:
-    signals += [("CAR_GAS", "GAS_PEDAL_2", 0),
```

Comment:
@energee so `GAS_PEDAL2` is not needed?

Reply:
What has now become ["GAS_PEDAL_2"]["CAR_GAS"] was originally the signal we used to determine "user pressing throttle". Some cars don't have the "GAS_PEDAL_2" message, so for these cars we started using another signal on the "POWERTRAIN_DATA" message labeled ["PEDAL_GAS"] which outputs the same data. 

<img width="532" alt="screen shot 2018-08-27 at 7 09 57 pm" src="https://user-images.githubusercontent.com/842240/44691677-a0a1ab00-aa2d-11e8-969e-b028b1ee8abd.png">

The interesting thing is that all hondas have the same powertrain_data message so it seems like a better place to read from and allows us to simplify carstate a bit.

Reply:
makes sense!

Reply:
idts

Reply:
Yes, but I don't think car_gas is used, we really just want the pedal_gas value. Maybe this car_gas value should be removed? In Toyota CS, car_gas is set to pedal_gas which isn't very accurate. The only other place I see it being used is in plant.py, and I don't know if its necessary there.